.eslintignore

```diff
@@ -1,4 +1,5 @@
 examples/
+test/e2e/testdata/**/**
```

Comment:
Need to ignore linting for the test configuration data otherwise the HTTP recordings won't precisely match.